CveXplore/database/maintenance/Sources_process.py

```diff
@@ -1024,6 +1024,71 @@ def populate(self, **kwargs):
         return self.update()
 
 
+class EPSSDownloads(CSVFileHandler):
+    def __init__(self):
+        self.feed_type = "EPSS"
+        self.delimiter = ","
+        super().__init__(self.feed_type, self.delimiter)
+
+        self.cve_collection = "cves"
+        self.feed_url = Configuration.getFeedURL(self.feed_type.lower())
+        self.logger = logging.getLogger(__name__)
+        self.is_update = True
+
+    @staticmethod
+    def process_epss_item(item=None):
+        if item is None:
+            return None
+
+        epss = {
+            "id": item[0],
+            "epss": item[1],
+            "epssMetric": {
+                "percentile": item[2],
+                "lastModified": date.strftime('%Y-%m-%dT%H:%M:%SZ')
+            }
+        }
+
+        return epss
+
+    def process_item(self, item):
+        # overwrite the feed_type, so that the correct collection is used
+        self.feed_type = self.cve_collection
```

Comment:
I don't know if I can do something better than this. I don't like the fact that I overwrite the feed_type, after the  download phase is done.

Reply:
Testing this PR at the moment; will perform some changes (if any) in the PR. Looking at this particular comment I am not convinced that overwriting the self.feed_type has anything to do with the items getting into the correct collection. I believe the feed_type is used when writing the lastModified values into the database.

Reply:
I don't think so, if you look at the insertion function from the DownloadHelper class -> https://github.com/cve-search/CveXplore/blob/master/CveXplore/database/maintenance/DownloadHandler.py#L157-L172 . You can see that self.feed_type is used for insertion into mongo. Specifying the collection in DatabaseAction is useless I think.

Reply:
You are correct; the collection specification in DatabaseAction is not used; I'll have to have a closer look!